### PR TITLE
Disable create slide and filter buttons while logged out

### PR DESCRIPTION
**Pull request by elmaxe**
I have several times tried to create slides when I unknowingly was logged out. Since there was no indicator it took me a while to realise that I had to log in. I therefore made those buttons disabled while not logged in. Also added an error text telling you to log in.

I also fixed the login/logout button styling on mobile devices. It no longer stretches across the whole screen from left to right, like it does since #79

### CODE REVIEW

File: client/views/addslides.html [hunk 1]
```diff
@@ -24,7 +24,12 @@
 				</div>
 				<div class="row">
 					<div class="input-field col s12">
+						{{#if currentUser}}
 						<button value="Create" class="createSlide btn">Create slide</button>
+						{{else}}
+						<button value="Create" class="createSlide btn disabled">Create slide</button>
+						<p class="error">Please log in to create a slide</p>
```
Comment (bullfest) [inline client/views/addslides.html]: `log in` here could be a link

File: client/views/filters.html [hunk 2]
```diff
@@ -9,7 +9,12 @@
 						<label class="active" for="filterbytags">Tags</label>
 					</div>
 					<div class="input-field col s12 m3">
+						{{#if currentUser}}
 						<button class="btn">Add tag</button>
+						{{else}}
+						<button class="btn disabled">Add tag</button>
+						<p class="error">Please log in to add filters</p>
```
Comment (bullfest) [inline client/views/filters.html]: And same here

Comment (elmaxe) [reply client/views/filters.html]: Login is not a link in the already existing messages, but sure

File: client/scripts/uploader.js [hunk 3]
```diff
@@ -4,6 +4,18 @@ var send_external_img = function(obj, slide, id){
 	$("." + id + "link").val("")
 }
 
+function getLoginUrl() {
+	console.log("ADHWHADAIDW")
```
Comment (bullfest) [inline client/scripts/uploader.js]: Loggylogg

File: client/scripts/uploader.js [hunk 4]
```diff
@@ -4,6 +4,18 @@ var send_external_img = function(obj, slide, id){
...
+		var callback = ""
+		callback += location.protocol + "//"
```
Comment (bullfest) [inline client/scripts/uploader.js]: Är det inte `://`?

File: client/scripts/uploader.js [hunk 5]
```diff
@@ -4,6 +4,18 @@ var send_external_img = function(obj, slide, id){
...
+		callback += location.hostname
+		if(location.port) {
+			callback += ":" + location.port
+		}
+		callback += "/login/"
```
Comment (bullfest) [inline client/scripts/uploader.js]: Lös det här hela lite elegant med en template literal istället 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: client/scripts/uploader.js [hunk 6]
```diff
@@ -4,6 +4,18 @@ var send_external_img = function(obj, slide, id){
...
+		return "http://login2.datasektionen.se/login?callback=" + callback
```
Comment (bullfest) [inline client/scripts/uploader.js]: Kan ändra till bara `login` istället för `login2` (det är legacy)

File: client/scripts/uploader.js (hunk 4 above)
Comment (elmaxe) [reply client/scripts/uploader.js]: Denna funktion är kopierad från `dashboard.js` så har ingen aning

File: client/scripts/uploader.js (hunk 6 above)
Comment (elmaxe) [reply client/scripts/uploader.js]: Som ovan, kopierade funktionen så har ingen aning om smådetaljerna

File: client/scripts/uploader.js (hunk 3 above)
Comment (elmaxe) [reply client/scripts/uploader.js]: rip

File: client/scripts/uploader.js (hunk 4 above)
Comment (elmaxe) [reply client/scripts/uploader.js]: Nej, location.protocol blir `http:` när jag printar i konsolen

Comment (bullfest) [reply client/scripts/uploader.js]: Där ser man...

**Pull request merged**